docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/AddGameOverNetwork.md

```diff
@@ -0,0 +1,116 @@
+```
+user@host:~$ scp
+usage: scp [-c cipher] [-D sftp_server_path] [-F ssh_config]
+           [-i identity_file] [-J destination] [-l limit] [-o ssh_option]
+           [-P port] [-S program] [-X sftp_option] source ... target
+user@host:~$
+```
+If you have trouble running these commands please consult the relevant documentation on installing ssh **client** software on your operating system.
+
+### Game Files
+This guide assumes that you are transferring **compiled** game binaries to the Arcade Machine. The Arcade Machine uses a Raspberry Pi, which has an ARM processor. When compiling your games make sure you **cross compile** for arm32. If your game is packaged as a .deb file you can install it connecting to the Arcade Machine via ssh (```ssh deakin@10.42.0.1```) and then running ```sudo dpkg -i myGame.deb``` to install it locally.
+
+### The "authenticity of host" warning message
+If this is the first time that you are connecting to the Arcade Machine you might see a prompt similar to the one below saying that authenticity of the host can't be established. **In this specific scenario** it's safe to type "yes" and continue:
+
+```
+user@host:~$ ssh user@arcademachine.local
+
+The authenticity of host '<host>' can't be established.
+ECDSA key fingerprint is    SHA256:<SomeHash>.
+Are you sure you want to continue connecting (yes/no)?
+```
+
+
+## Connecting To Arcade Machine WiFi Access Point
+Find the SSID of the WiFi access point corresponding to the Arcade Machine you want to copy games to. When prompted enter the WiFi password. By default this is ```GamesAreFun```. 
+
+The Arcade Machine has no [DHCP](https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol) service, so you will need to manually set a static IP address once you have connected to the network. You will need the following network settings:
+
+The Arcade Machine subnet is ```10.42.0.1 /24``` Your network configuration should therefore look like:
+```
+IP Address:  10.42.0.2
+Subnet Mask: 255.255.255.0 
+Gateway:     10.42.0.1
+```
+
+**Note:** If another student is concurrently accessing the Arcade Machine you will need to choose another IP Address, or you won't be able to use the network!
+
+Verify connectivity by pinging the Arcade Machine's IP address:
+```
+user@host:~$ ping 10.42.0.1
+PING 10.42.0.1 (10.42.0.1) 56(84) bytes of data.
+64 bytes from 10.42.0.1: icmp_seq=1 ttl=58 time=43.0 ms
+64 bytes from 10.42.0.1: icmp_seq=2 ttl=58 time=47.0 ms
+64 bytes from 10.42.0.1: icmp_seq=3 ttl=58 time=40.8 ms
+64 bytes from 10.42.0.1: icmp_seq=4 ttl=58 time=41.1 ms
+64 bytes from 10.42.0.1: icmp_seq=5 ttl=58 time=40.6 ms
+^C
+--- 10.42.0.1 ping statistics ---
+5 packets transmitted, 5 received, 0% packet loss, time 4001ms
+user@host:~$ 
+```
```

Comment:
This all seems very complex from a users point of view and there's the potential that someone doesn't know how to set a static ip address for their operating system (e.g. the process of setting a static ip on windows is painfully long). I'm wondering if it might be easier if we run `dnsmasq` on the pi's to provide a dhcp server?

Reply:
This is a catch-22. On one hand setting a static IP address is an annoying process for some users, on the other hand we'd have to make _another_ configuration change to the Arcade Machine implement a DHCP server.  I'm personally disinclined adding another service to manage if we can at all avoid it, even if this comes at the cost of some user friendliness.

There's also a small amount of system resources overhead required to run the DHCP server which i'd rather not invest on the already constrained hardware.

I know that dnsmasq can be configured to operate in proxy mode, but we'd also need to make sure this doesn't interfere with how the devices operate on the eduroam network. 

Ultimately I don't think that setting static IP addresses should be outside the realm of capability of a user base that is consisting primarily of computer science students.

Reply:
Yeap that's fair.